config/routes.rb

```diff
@@ -235,6 +235,8 @@
     end
     collection do
       post 'notify_instructors'
+      post 'reply'
```

Comment:
I'm confused about this. Is it used by the frontend?

Reply:
Is was meant for replying to an email, since i was referencing the ticket system, but I have removed it as the front end does not use it currently